packages/react-examples/src/react-link/Link/Link.Basic.Example.tsx

```diff
@@ -10,8 +10,16 @@ export const LinkBasicExample: React.FunctionComponent = () => {
         <Link href="https://developer.microsoft.com/en-us/fluentui#/controls/web/link">
           it renders as an anchor tag.
         </Link>{' '}
-        Without an href, <Link>the link is rendered as a button</Link>. You can also use the disabled attribute to
-        create a{' '}
+        Without an href,
```

Comment:
Need to add a space here
```suggestion
        Without an href,{' '}
```

Reply:
shouldn't that be handled by prettier ? 🤔

Reply:
Depends on how you did the line breaks before running prettier. If you start with the element on the same line as the preceding text, Prettier correctly adds the space when breaking the lines. But if you already added a line break before the element, Prettier won't add a space.